moveit_kinematics/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
 endif()
 
 find_package(Boost)
+find_package(ompl REQUIRED)
```

Comment:
This is an unfortunate mixing of dependencies - previously the ompl dependency could be removed by simply removing the moveit_ompl_interface package. This will require anyone using IK in MoveIt! (everyone) to need OMPL. Is there a way to use the NN without OMPL?

Reply:
Yes, I can copy over the relevant files from OMPL.

Reply:
Would it make things better if we use flann here?
Neither copying files nor a new dependency on OMPL sound too appealing.

That being said I wouldn't oppose the OMPL dependency here myself.
De-facto `moveit` depends on `moveit_planners` which depends on `moveit_planners_ompl` which depends on `ompl`. So rosdep already shows this "hard" dependency.
Even though of course you can ignore it if you know what you're doing.